packages/core/src/mixins/as-text-input.scss

```diff
@@ -0,0 +1,44 @@
+@use '../helpers';
+@use './with-inner-focus' as mixins;
```

Comment:
[Minor] Why not still use mixins as before:

```suggestion
@use '../mixins';
```

Reply:
Because it creates a bidirectional import path, or a loop.
`as-text-input` imports `mixins/index`, while `mixins/index` imports `as-text-input`.
Sass might be clever enough to make it work, but it's bad practice 🙊

Reply:
Actually it doesn't work
![image](https://user-images.githubusercontent.com/18623773/105179092-c5c47600-5b20-11eb-8dae-5e9a8e94dfb4.png)

Reply:
Yeah, it's OK - it was a minor comment, I do not require actions on these 😄

Thanks for checking though!